presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSpatialJoins.java

```diff
@@ -61,35 +45,7 @@
 
     public TestSpatialJoins()
     {
-        super(() -> createQueryRunner());
-    }
-
-    private static DistributedQueryRunner createQueryRunner()
-            throws Exception
-    {
-        DistributedQueryRunner queryRunner = new DistributedQueryRunner(testSessionBuilder()
-                .setSource(TestSpatialJoins.class.getSimpleName())
-                .setCatalog("hive")
-                .setSchema("default")
-                .build(), 4);
-        queryRunner.installPlugin(new GeoPlugin());
-
-        File baseDir = queryRunner.getCoordinator().getBaseDataDir().resolve("hive_data").toFile();
-
-        HiveClientConfig hiveClientConfig = new HiveClientConfig();
-        HdfsConfiguration hdfsConfiguration = new HiveHdfsConfiguration(new HdfsConfigurationUpdater(hiveClientConfig));
-        HdfsEnvironment hdfsEnvironment = new HdfsEnvironment(hdfsConfiguration, hiveClientConfig, new NoHdfsAuthentication());
-
-        FileHiveMetastore metastore = new FileHiveMetastore(hdfsEnvironment, baseDir.toURI().toString(), "test");
-        metastore.createDatabase(Database.builder()
-                .setDatabaseName("default")
-                .setOwnerName("public")
-                .setOwnerType(PrincipalType.ROLE)
-                .build());
-        queryRunner.installPlugin(new HivePlugin("hive", Optional.of(metastore)));
-
-        queryRunner.createCatalog("hive", "hive");
-        return queryRunner;
+        super();
```

Comment:
A default constructor calling `super()` is unnecessary unless I am missing something here.